tests/unit/test_launchers.py

```diff
@@ -331,7 +331,6 @@ def test_firefox_install(
 
 
 class TestFennecLauncher(unittest.TestCase):
-
```

Comment:
Unintentional addition?

Reply:
It is intentional, as the new version of Black checks this.

Reply:
See https://github.com/mozilla/mozregression/actions/runs/4086181052/jobs/7045143766.